models.py

```diff
@@ -0,0 +1,29 @@
+# This encodes the different models
+
+def m1(expe,alphaG = .1,alphaN=.1,alphaC=.1,betaG=1,betaN=1):
+    """
+    This encodes RL model type 1
+    :param Expe: is the structure of the experiment through which M1 is run
+    :param alphaG: G learning rate
```

Comment:
If this code is intended to be used by others, specifying reasonable/allowed values and a bit more about what this means is helpful. How will it be used? How is it different from alpha_c?